deployment/GP02HYPM_Deploy.csv

```diff
@@ -97,3 +91,17 @@ SKQ201920S,,,,GP02HYPM-WFP03-01-FLORDL000,7,1,2019-09-26T00:05:00,,CGMGP-02HYPM-
 SKQ201920S,,,,GP02HYPM-WFP03-03-DOSTAL000,7,1,2019-09-26T00:05:00,,CGMGP-02HYPM-00007,CGVEH-GWFPGX-10403,CGINS-DOSTAL-01478,50.11518,-144.91612,,4063,4237,,,
 SKQ201920S,,,,GP02HYPM-WFP03-04-CTDPFL000,7,1,2019-09-26T00:05:00,,CGMGP-02HYPM-00007,CGVEH-GWFPGX-10403,CGINS-CTDPFL-00132,50.11518,-144.91612,,4063,4237,,,
 SKQ201920S,,,,GP02HYPM-WFP03-05-VEL3DL000,7,1,2019-09-26T00:05:00,,CGMGP-02HYPM-00007,CGVEH-GWFPGX-10403,CGINS-VEL3DL-01115,50.11518,-144.91612,,4063,4237,,,
+SKQ202111S,,,,GP02HYPM-MPM01-02-ZPLSGA009,8,1,2021-07-18T00:00:00,,CGMGP-02HYPM-00008,,CGINS-ZPLSGA-55092,50.07,-144.8,,150,4219,upward-looking,,
+SKQ202111S,,,,GP02HYPM-MPM01-02-ZPLSGA010,8,1,2021-07-18T00:00:00,,CGMGP-02HYPM-00008,,CGINS-ZPLSGA-55093,50.07,-144.8,,150,4219,downward-looking,,
+SKQ202111S,,,,GP02HYPM-RIM01-00-SIOENG000,8,1,2021-07-18T00:00:00,,CGMGP-02HYPM-00008,,CGCON-SIOCON-12371,50.07,-144.8,,4073,4219,,,
+SKQ202111S,,,,GP02HYPM-RIM01-02-CTDMOG039,8,1,2021-07-18T00:00:00,,CGMGP-02HYPM-00008,,CGINS-CTDMOG-13430,50.07,-144.8,,155,4219,,,
+SKQ202111S,,,,GP02HYPM-WFP02-00-WFPENG000,8,1,2021-07-18T00:00:00,,CGMGP-02HYPM-00008,CGVEH-GWFPGX-10402,CGCON-GWFPGX-10402,50.07,-144.8,,2090,4219,,,
+SKQ202111S,,,,GP02HYPM-WFP02-01-FLORDL000,8,1,2021-07-18T00:00:00,,CGMGP-02HYPM-00008,CGVEH-GWFPGX-10402,CGINS-FLORDL-03283,50.07,-144.8,,2090,4219,,,
+SKQ202111S,,,,GP02HYPM-WFP02-03-DOSTAL000,8,1,2021-07-18T00:00:00,,CGMGP-02HYPM-00008,CGVEH-GWFPGX-10402,CGINS-DOSTAL-01480,50.07,-144.8,,2090,4219,,,
+SKQ202111S,,,,GP02HYPM-WFP02-04-CTDPFL000,8,1,2021-07-18T00:00:00,,CGMGP-02HYPM-00008,CGVEH-GWFPGX-10402,CGINS-CTDPFL-00134,50.07,-144.8,,2090,4219,,,
+SKQ202111S,,,,GP02HYPM-WFP02-05-VEL3DL000,8,1,2021-07-18T00:00:00,,CGMGP-02HYPM-00008,CGVEH-GWFPGX-10402,CGINS-VEL3DL-01108,50.07,-144.8,,2090,4219,,,
+SKQ202111S,,,,GP02HYPM-WFP03-00-WFPENG000,8,1,2021-07-18T00:00:00,,CGMGP-02HYPM-00008,CGVEH-GWFPGX-93602,CGCON-GWFPGX-93602,50.07,-144.8,,4063,4219,,,
+SKQ202111S,,,,GP02HYPM-WFP03-01-FLORDL000,8,1,2021-07-18T00:00:00,,CGMGP-02HYPM-00008,CGVEH-GWFPGX-93602,CGINS-FLORDL-02736,50.07,-144.8,,4063,4219,,,
+SKQ202111S,,,,GP02HYPM-WFP03-03-DOSTAL000,8,1,2021-07-18T00:00:00,,CGMGP-02HYPM-00008,CGVEH-GWFPGX-93602,CGINS-DOSTAL-01086,50.07,-144.8,,4063,4219,,,
```

Comment:
Based on the cals in 12936-02 on Vault, shouldn't this be CGINS-DOSTAL-01086? Or have we just not received/posted the new cals for this WFP yet?

Reply:
Yes - you are correct!  This has been updated.  Need to remember to add this sensor to sensor bulk load.

Reply:
After further investigation we believe 1086 is correct and that the vendor calibration document is incorrect.  Will reach out to Aanderaa to confirm & get a corrected vendor cal doc.  Will also visually confirm SN once things are unloaded in Seward.

Reply:
Looks good